### PR TITLE
Fix bad links

### PR DESCRIPTION
**Pull request by esthermmoturi**
# Description

Fixes bad links highlighted by  bot ran on Sep 15th 2024.

```
Run ./.github/scripts/muffet.sh
http://localhost:1313/hosting/4.x/app-developer/
	404	http://localhost:1313/apps/tutorials/local-setup/privacy.error.png
http://localhost:1313/building/guides/debugging/secure-sharing-of-developer-instance/
	404	http://localhost:1313/apps/guides/debugging/images/SSH.tunnel.diagram.svg
	404	http://localhost:1313/apps/guides/debugging/images/ssh.ports.svg
http://localhost:1313/contribute/code/
	502	https://alpha.dev.medicmobile.org/
http://localhost:1313/contribute/medic/onboarding/technical-resources/
	403	https://www.udacity.com/course/asynchronous-javascript-requests--ud109
http://localhost:1313/building/guides/data/analytics/couch2pg-to-cht-sync-migration/
	404	http://localhost:1313/building/guides/data/analytics/couch2pg-to-cht-sync-migration/(/building/guides/data/analytics/building-dbt-models/)
	404	http://localhost:1313/building/guides/data/analytics/couch2pg-to-cht-sync-migration/(/building/guides/data/analytics/production/)
	404	http://localhost:1313/building/guides/data/analytics/couch2pg-to-cht-sync-migration/(/building/guides/data/analytics/setup/)
	404	http://localhost:1313/building/guides/data/analytics/couch2pg-to-cht-sync-migration/(/hosting/monitoring/setup/)
http://localhost:1313/hosting/3.x/app-developer/
	404	http://localhost:1313/apps/tutorials/local-setup/privacy.error.png
Error: Process completed with exit code 1.
```

### CODE REVIEW

**Comment by mrjones-plip** [comment]
@esthermmoturi - we should always use text instead of screenshots in tickets - this allows us to search for the content of the error messages.

thanks!

**Comment by esthermmoturi** [comment]
Hey @mrjones-plip , not sure the image you are referring to, is it what I have placed in the description? If yes, it's a text block.

**Comment by mrjones-plip** [comment]
Correct! It is a text block now, but was an image until I edited it ;)

image

**Comment by esthermmoturi** [comment]
My bad @mrjones-plip! Thanks. I will use text moving forward.

**Review by andrablaj — COMMENTED** [review]
@esthermmoturi I fixed the cht-sync doc related links last week in [this PR](https://github.com/medic/cht-docs/pull/1529/files), in addition to the removal of CHT Pipeline mentions and they have somehow been overwritten. Any chance to apply the changes in my PR to this PR too?

**Comment by esthermmoturi** [comment]
Hey @andrablaj , please confirm that it's now okay.

**Comment by andrablaj** [comment]
The problem seems to be in [this commit](https://github.com/medic/cht-docs/pull/1537/files#diff-1c4421545ac5dd7120937c0446d2af68c0f4c94b497cfa8953a3b67c18462fc9) where the changes I pushed (probably just before you messaged the team not to merge anything that day 😬) got overwritten. I can open a new PR if it makes is easier.

**Comment by esthermmoturi** [comment]
@andrablaj Let me try merging with your branch. Give me a minute.

File: content/en/building/guides/data/analytics/couch2pg-to-cht-sync-migration.md [hunk 1]
```diff
@@ -16,16 +16,18 @@ This page outlines guidelines for migrating from [couch2pg](https://github.com/m
...
+1. **Set up monitoring and alerting**: [Setup CHT Watchdog]({{< relref "hosting/monitoring/setup" >}}) to monitor the running of CHT Sync and set up alerts for any failures.
+1. **Optimize**: Once the dbt models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [dbt models guide]({{< relref "building/guides/data/analytics/building-dbt-models" >}}) will help you understand how to optimize the models.
+1. **Set up monitoring and alerting**: [Setup CHT Watchdog]({{< relref "hosting/monitoring/setup" >}}) to monitor the running of CHT Sync and CHT Pipeline and set up alerts for any failures.
```
Comment (andrablaj) [inline content/en/building/guides/data/analytics/couch2pg-to-cht-sync-migration.md:32]:
```suggestion
```

File: content/en/building/guides/data/analytics/couch2pg-to-cht-sync-migration.md [hunk 2]
```diff
@@ -16,16 +16,18 @@ This page outlines guidelines for migrating from [couch2pg](https://github.com/m
...
+1. **Optimize**: Once the dbt models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [dbt models guide]({{< relref "building/guides/data/analytics/building-dbt-models" >}}) will help you understand how to optimize the models.
+1. **Set up monitoring and alerting**: [Setup CHT Watchdog]({{< relref "hosting/monitoring/setup" >}}) to monitor the running of CHT Sync and set up alerts for any failures.
+1. **Optimize**: Once the dbt models are working as expected and the dashboards display the expected data, optimize the models to improve performance. This may involve restructuring the models, adding indexes, or making other adjustments to improve the speed and efficiency of the models. Having a look at the [dbt models guide]({{< relref "building/guides/data/analytics/building-dbt-models" >}}) will help you understand how to optimize the models.
```
Comment (andrablaj) [inline content/en/building/guides/data/analytics/couch2pg-to-cht-sync-migration.md:31]:
```suggestion
```

**Review by andrablaj — CHANGES_REQUESTED** [review]
For some reason a paragraph got duplicated. Otherwise all good!

**Review by andrablaj — APPROVED** [review]
Looks good, thanks!

**Pull request merged**